test-suite/src/order_by.rs

```diff
@@ -150,4 +150,60 @@ CREATE TABLE Test (
         "SELECT * FROM Test ORDER BY id NULLS FIRST",
         Err(TranslateError::OrderByNullsFirstOrLastNotSupported.into())
     );
+    test!(
+        // ORDER BY aliases
+        "SELECT id AS C1, num AS C2 FROM Test ORDER BY C1 ASC, C2 DESC",
+        Ok(select!(
+            C1  | C2
+            I64 | I64;
+            1     9;
+            1     2;
+            3     4;
+            4     7
+        ))
+    );
+    test!(
+        // original column_names still work even if aliases were used at SELECT clause
+        "SELECT id AS C1, num AS C2 FROM Test ORDER BY id ASC, num DESC",
+        Ok(select!(
+            C1  | C2
+            I64 | I64;
+            1     9;
+            1     2;
+            3     4;
+            4     7
+        ))
+    );
```

Comment:
Thanks to @zmrdltl 
New `test!` macro is now available which can specify test name.
This test already has a good title, so.. how about to start using the new `test!`?

```rust
test! {
    name: "original column_names still work even if aliases were used at SELECT clause",
    sql: "
        SELECT id AS C1, num AS C2
        FROM Test
        ORDER BY id ASC, num DESC
    ",
    expected:
        Ok(select!(
            C1  | C2
            I64 | I64;
            1     9;
            1     2;
            3     4;
            4     7
        ))
};
```